### PR TITLE
[REVIEW] Add BlazingSQL version of query 09

### PR DESCRIPTION
**Pull request by Christian8491**
This PR adds the version of "query" 09 using `BlazingSQL`

### CODE REVIEW

**Comment by Christian8491** [comment]
It shows **Correctness Assertion True** at scale factor **1K**, using **dgx201**

```
(tpcx-bb) christianc@rl-dgx2-d17-u05-rapids-dgx201:~/tpcx-bb/tpcx_bb/queries/q09$ python tpcx_bb_query_09_sql.py --data_dir=/raid/tpcx-bb/sf1000/parquet_2gb/ --verify_results --verify_dir=/mnt/weka/nicholasb/tpcx-bb/validation/sf1000/ --cluster_host=10.150.162.27
Using default arguments
Connected to tcp://10.150.162.27:8786
WARNING: Count not make directory
WARNING:root:WARNING: Count not make directory
WARNING: Count not make directory
WARNING:root:WARNING: Count not make directory
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.protobuf.UnsafeUtil (file:/home/christianc/miniconda3/envs/tpcx-bb/lib/blazingsql-algebra.jar) to field java.nio.Buffer.address
WARNING: Please consider reporting this to the maintainers of com.google.protobuf.UnsafeUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
BlazingContext ready
Standard ETL Query
/home/christianc/miniconda3/envs/tpcx-bb/lib/python3.7/site-packages/pandas/util/__init__.py:12: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.
  import pandas.util.testing
Correctness Assertion True
Must pass a sheet and tab name to use Google Sheets automation
```

Do you think this PR should change to the **REVIEW** state?

File: tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+# limitations under the License.
+#
+
+import sys
+
+
+from blazingsql import BlazingContext
+from xbb_tools.cluster_startup import attach_to_cluster
+from dask_cuda import LocalCUDACluster
+from dask.distributed import Client
```
Comment (beckernick) [inline tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py:24]: Looks like we can remove these unused imports

File: tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+
```
Comment (beckernick) [inline tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py:19]:
```suggestion

```

File: tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                ca1.ca_country = 'United States'
+                AND ca1.ca_state IN ('WI', 'MO', 'WV')
+                AND 50 <= ss1.ss_net_profit
+                AND ss1.ss_net_profit <= 25000
+            )
+        )
+    """
+
+    result = bc.sql(query)
+    result = result.rename(columns={"EXPR$0": "sum(ss_quantity)"})
```
Comment (beckernick) [inline tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py:111]: I believe this still introduces an additional copy versus `result.columns = [...]` with Dask, so if possible we would want to use `.columns`

**Comment by beckernick** [comment]
@Christian8491 , when you believe a PR is ready for review, can you:
- Update the title to switch [WIP] to [REVIEW]
- Update the "Label" (on the right side of this Github page) to the category "Ready for Review"

Thanks!

File: tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py (hunk 1 above)
Comment (Christian8491) [reply tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py]: Removed.

File: tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py (hunk 2 above)
Comment (Christian8491) [reply tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py:19]: Line 19 removed.

File: tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py (hunk 3 above)
Comment (Christian8491) [reply tpcx_bb/queries/q09/tpcx_bb_query_09_sql.py]: Updated as

**result.columns = ["sum(ss_quantity)"]**

**Comment by Christian8491** [comment]
After applying these changes (and the separate values as Spark version) I just tested again at sf **1K** and got the **Correctness Assertion True** message again using the **dgx201**, so updating this PR to **REVIEW**.

It looks like I don't have access to update the "Label" to the category "Ready for Review"

**Comment by beckernick** [comment]
> It looks like I don't have access to update the "Label" to the category "Ready for Review"

Will figure this out

**Pull request merged**